src/lib/metricsMgr/MetricsManager.cpp

```diff
@@ -216,6 +227,57 @@ void MetricsManager::reset(void)
 
 /* ****************************************************************************
 *
+* metricsRender - 
+*/
+static std::string metricsRender(std::map<std::string, uint64_t>& metricsMap)
+{
+  std::map<std::string, uint64_t>::iterator  it;
+  uint64_t                                   incomingTransactions = 0;
+  uint64_t                                   totalServiceTime     = 0;
+  int                                        metricsAdded         = 0;
```

Comment:
(Out-topic regarding compiler: it could be our fault... we have weird things happening at built, see https://github.com/telefonicaid/fiware-orion/issues/2726)